google-cloud-spanner/src/test/java/com/google/cloud/spanner/OptionsTest.java

```diff
@@ -510,4 +517,68 @@ public void testTransactionOptionsPriorityEquality() {
     assertNotEquals(options2, options3);
     assertNotEquals(options2, options4);
   }
+
+  @Test
+  public void updateOptTest() {
+    String tag = "app=spanner,env=test";
+    Options opts = Options.fromUpdateOptions(Options.tag(tag));
+
+    assertThat(opts.toString()).isEqualTo("tag: " + tag + " ");
```

Comment:
Use of Truth here feels gratuitous. assertEquals and assertTrue are simple and well understood.

Reply:
Done.